sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/JwtVendor.java

```diff
@@ -42,6 +42,16 @@ static JsonWebKey getDefaultJsonWebKey() {
         return jwk;
     }
 
+    static JsonWebKey getHS256WebKey() {
```

Comment:
Will this method be used outside of test scenarios?

Reply:
It could be if people specified it but it is not intended to be used for anything other than testing.